### PR TITLE
Log4j FAQ link, text and search string adjustments

### PR DESCRIPTION
**Pull request by browkin**
- Made Links prettier
- Text adjustments
- Fixed Search String

Suggestions by @MikeMcC399 & @Ein-Tim 

**OLD:**

log4jOLDde
log4jOLDen

**NEW:**

log4NEWde
log4jNEWEN

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -92,13 +92,13 @@
...
-                            "In response to this recent discovery and in order to minimize potential impacts the project teams of SAP and Deutsche Telekom have joined the worldwide effort and taken action. This also includes the update to 2.16.0 of the affected log4j library leveraged by the server component. There is no need to take action by the user, both the Corona-Warn-App and the backend systems are safe.",
-                            "There is a series of pull requests related to this issue. They can be found in the repository of our open source project: <a href='https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j' target='_blank' rel='noopener'>https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j</a>"
+                            "There is a series of pull requests related to this issue. They can be found in the repository of our open source project: <a href='https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j+in%3Atitle+merged%3A%3E2021-12-01' target='_blank' rel='noopener'>Github</a>."
```
Comment (Ein-Tim) [inline src/data/faq.json:101]:
```suggestion
                            "There is a series of pull requests related to this issue. They can be found in the repository of our open source project: <a href='https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j+in%3Atitle+merged%3A%3E2021-12-01' target='_blank' rel='noopener'>GitHub</a>."
```

File: src/data/faq_de.json [hunk 2]
```diff
@@ -92,14 +92,14 @@
...
-                            "Als Reaktion auf diese jüngste Entdeckung und um mögliche Auswirkungen zu minimieren, haben sich die Projektteams von SAP und Deutsche Telekom den weltweiten Bemühungen angeschlossen und Maßnahmen ergriffen. Dazu gehört auch ein Server-Update auf 2.16.0 der betroffenen log4j-Bibliothek. Der Anwender muss hier nicht aktiv werden, die Corona-Warn-App und die Backend-Systeme sind sicher.",
-                            "Es gibt eine Reihe von Pull-Requests, die sich auf dieses Problem beziehen. Sie können im Repository unseres Open-Source-Projekts gefunden werden: <a href='https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j' target='_blank' rel='noopener'>https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j</a>"
+                            "Es gibt eine Reihe von Pull-Requests, die sich auf dieses Problem beziehen. Sie können im Repository unseres Open-Source-Projekts gefunden werden: <a href='https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j+in%3Atitle+merged%3A%3E2021-12-01' target='_blank' rel='noopener'>Github</a>."
```
Comment (Ein-Tim) [inline src/data/faq_de.json:102]:
```suggestion
                            "Es gibt eine Reihe von Pull-Requests, die sich auf dieses Problem beziehen. Sie können im Repository unseres Open-Source-Projekts gefunden werden: <a href='https://github.com/corona-warn-app/cwa-server/pulls?q=is%3Apr+log4j+in%3Atitle+merged%3A%3E2021-12-01' target='_blank' rel='noopener'>GitHub</a>."
```

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
Replace "Github" with "GitHub"

File: src/data/faq_de.json (hunk 2 above)
Comment (browkin) [reply src/data/faq_de.json:102]: @Ein-Tim Thanks missed that one!

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for the review.

**Review by MikeMcC399 — APPROVED** [review]
This looks fine to me now. The text is generic and will not need to be changed if there are any new events concerning Log4j.

Thanks for including my search string suggestion. That filters out non-relevant PRs.

It looks ready to merge!

**Comment by dsarkar** [comment]
@MikeMcC399 Thanks for the review.

**Pull request merged**